### ISSUE TITLE
Points for Improvement

### ISSUE THREAD

**Issue by qianpeisheng** [qianpeisheng/indo#2]
1. Validation loss and accuracy are inconsistent (both increases during training).
2. Use full bert (currently bert lite).
3. Check the effectiveness of using first [CLS] token to classify if poi/street exists, try to scale the loss ratio/ try other loss fn, or other training techniques.
4. Inspect the training log, e.g., the current `street_end_epoch` accuracy is relatively low. 
3. Train more epochs with full training data (the submitted one is epoch 10, 80% training data).
4. save checkpoints (Now yes but not the best practice) and model configs (Now no).

Notes: training takes ~12 min/epoch on single V100.
